### PR TITLE
SG-21170-ROUND2 [ZD to AKN] Update links referenced from code to the old support site to point to AKN

### PR DESCRIPTION
**Pull request by 000paradox000**
This PR replaces ZD urls to AKN and developer urls

### CODE REVIEW

File: info.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ configuration:
         type: str
         description: Link to a ShotGrid Support article detailing how to
                      configure Software entities for a site.
-        default_value: "https://support.shotgunsoftware.com/entries/95443887#Setting%20up%20Application%20Paths"
+        default_value: "https://developer.shotgridsoftware.com/1b9c259a/?title=Launch+App#Setting%20up%20Application%20Paths"
```
Comment (mathurf) [inline info.yml:22]: This is about the Software entities page: https://developer.shotgridsoftware.com/8085533c/?title=Shotgun+Integrations+Admin+Guide#configuring-software-launches

**Review by mathurf — CHANGES_REQUESTED** [review]
Please fix the URL

File: info.yml (hunk 1 above)
Comment (000paradox000) [reply info.yml]: Done

**Review by mathurf — APPROVED** [review]
LGTM!

**Pull request merged**